docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -0,0 +1,248 @@
+
+This model will allow us to understand and quantify the impact of different combinations of preventive and treatment strategies on wasting and wasting attributable DALYs at the national level. 
+
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+
+The primary objective of this project is to answer the following question: what is the cost and impact of combinations of preventive and therapeutic strategies for reducing overall wasting prevalence and eliminating SAM in Ethiopia? 
+We will use data from the 2019 GBD and published literature to inform the parameters for our simulation. We will simulate the changes in MAM and SAM disease incidence, prevalence and mortality from 2022 to 2027 in response to a combination of preventive and therapeutic treatment interventions in Ethiopia.
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal diagram
+------------------
+
+ .. note::
+    link to DAGs page
+    use round circles with DAGs
+
+**Outcome (O)**:
+
+
+
+**Most proximal determinant/exposure (E)**:
+
+
+
+**Confounders (C)**:
+
+
+
+**Effect modifiers**:
+
+
+**Mediators (M)**:
+
+
+.. _3.2:
+
+3.2 Effect sizes
+----------------
+
+
+
+4.0 Intervention
+++++++++++++++++
+
+Historically, prevention research has primarily focused on stunting, and, as a research outcome, wasting has been considered primarily within the context of humanitarian emergencies. Although the volume of studies related to wasting prevention through direct and indirect health-care sector areas has increased in recent years, this evidence base is mixed and often inconclusive8. We reviewed the literature from the recent Keats et al 2021 update of effective interventions to address maternal and child malnutrition and selected interventions that have moderate or strong evidence for implementation5. We selected interventions that:  
```

Comment:
Remove citation numbers left over from word doc version - inclusive8 and implentation5